test/core/utils/node-serializer.js

```diff
@@ -91,14 +91,19 @@ describe('nodeSerializer', () => {
     });
 
     it('skips computing props turned off with runOptions', () => {
-      const dqElm = new DqElement(fixture);
-
       const throws = () => {
         throw new Error('Should not be called');
       };
-      Object.defineProperty(dqElm, 'selector', { get: throws });
-      Object.defineProperty(dqElm, 'ancestry', { get: throws });
-      Object.defineProperty(dqElm, 'xpath', { get: throws });
+
+      const dqElm = new DqElement(
+        fixture,
+        {},
+        {
+          selector: { get: throws },
```

Comment:
With memoize this style of define would propagate to all tests and cause issues. Since `DqElement` takes the spec as truth, just passed the functions in as the spec.